src/components/ResultItem/ResultItem.tsx

```diff
@@ -1,17 +1,32 @@
 import React from 'react';
 
+type meta = {
+  type?: string;
+  date?: Date;
+  lastUpdated?: Date;
+};
+
 type ResultItemProps = {
   className?: string;
   href: string;
-  meta?: string;
+  meta?: meta;
   text?: string;
   title: string;
 };
 
-export const ResultItem = ({ meta, title, href, text }: ResultItemProps) => (
+export const ResultItem = ({
+  meta: { type, date, lastUpdated },
+  title,
+  href,
+  text
+}: ResultItemProps) => (
   <div className="result-item">
     <div className="result-item__meta">
-      <span className="result-item__meta-type">{meta}</span>
+      <span className="result-item__meta-type">{type}</span>
+      <span className="result-item__meta-type--date">{date}</span>
+      <span className="result-item__meta-type--last-updated">
+        Update {lastUpdated}
```

Comment:
These props will be accessed with `meta.type`, `meta.date` and `meta.lastUpdated` (I think)

Reply:
I don't think so as it's giving me error if I add meta.